src/refuse/_refactor.py

```diff
@@ -32,6 +32,7 @@
 # +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 import os
+import sys
```

Comment:
Super pedantic; I'd be tempted to import AFTER ctypes to be more Pythnonic (alphabetical sorted imports, note existing `import os` could move too)

Reply:
Sure, I'd be happy to do either or both of those small modifications.  @s-m-e do you have any thoughts?